docs/user/pixel_observations.md

```diff
@@ -0,0 +1,122 @@
+## Using Image Observations
+Although most environments return observations that somehow describe the physical state of the agent and environment, others return observations that contain spatial data instead. As an example, consider the `MemorizeDigits-v0` gym environment, where the agent must memorize what each digit looks like using RGB observations such as:
+
+```
+  ---------------------------
+ |                           |
+ |         ******            |
+ |         ******            |
+ |       **      **          |
+ |       **      **          |
+ |               **          |
+ |               **          |
+ |           ****            |
+ |           ****            |
+ |       ****                |
+ |       ****                |
+ |       **********          |
+ |       **********          |
+ |                           |
+  ---------------------------
+```
+
+Image observations can be represented as tensors containing each individual pixel's RGB value. Typically these tensors have the shape `(HEIGHT, WIDTH, CHANNELS)` where CHANNELS denotes the number of channels. RGB images have 3 channels and grayscale images have 1 channel.
+
+This page will guide you through managing these image observations and environments.
+## Pixel Environments
+The following gym environments have been tested with garage and have examples written for them:
+ - `MemorizeDigits-v0` - see  `examples/tf/ppo_memorize_digits.py`
+ - `CubeCrash-v0`- see `examples/tf/trpo_cubecrash.py`
+ - `PongNoFrameskip-v4` - see `examples/tf/dqn_pong.py`
+
+ Gym environments that do not return pixel observations by default can be used with `garage.envs.wrappers.PixelObservationWrapper`, which overrides the default observation and instead returns a tensor of RGB values by making calls to `env.render(mode="rgb_array")`. See the section on this wrapper below for more on this.
+
+## Environment Wrappers
+
+Garage includes various environment wrappers that are particularly useful when working with pixel observations. You may use multiple wrappers simultaneously, though the order in which you use them matters. See the pixel-specific wrappers section for more details on this.
+
+### GarageEnv
+When wrapping image-based environments in `GarageEnv`, `is_image` should be set to `True`:
+```python
+env = GarageEnv(gym.make("MemorizeDigits-v0"), is_image=True)
+```
+This converts the `gym.spaces.Box` observation space to an `akro.Image` space, which then:
+- Tells CNN primitives in garage to normalize pixel values to between 0 and 1. This typically decreases training time, particularly in environments where observation values vary widely. Note that this is only done internally within the primitives, so calls to `env.step()` will still return raw pixel values.
```

Comment:
Do we store images as `int` in replay buffer and only normalize them when passing to policy, so as to save memory? If so, I think it's also worth mentioning as it is an important technique in processing spatial data.

Reply:
So we don't explicitly cast non-int values to ints for images before storing them in the buffer (though maybe we should). We assume that, if the input is an image, the environment will output ints. I'll mention that the user should make sure this is the case though, for the reasons you mentioned.